src/components/reusable/table/index.css

```diff
@@ -1,8 +1,8 @@
-.ant-table-row:nth-child(even)
+.ant-table-row:nth-child(odd)
```

Comment:
did u check the other areas where the table is used? is that working fine?

Reply:
Yes checked, 
it is working fine...